net/cloudflared/files/cloudflared.init

```diff
@@ -9,8 +9,17 @@ PROG="/usr/bin/cloudflared"
 
 append_param_arg() {
 	local value
+	# Use $1 as key, get value from uci
+	# If value empty, append $1 as param, $2 as value
 	config_get value "config" "$1" $2
-	[ -n "$value" ] && procd_append_param command "--$1" "$value"
+	# If value not empty, value as value
+	[ -n "$value" ] && if [ -n "$3" ]; then
+		# If has $3, append $3 as param
+		procd_append_param command "--$3" "$value"
```

Comment:
```suggestion
	[ -n "$value" ] && procd_append_param command "--${1//_/-}" "$value"
```

keep it simple

Reply:
Agree, I don't know what writing style should I use so I chose `readability` for default.

Reply:
Oh, it can written in that way!  
Learned this, thanks!